cmd/egress/cmd.go

```diff
@@ -36,7 +36,14 @@ func NewCmdValidateEgress() *cobra.Command {
 			ctx := context.TODO()
 
 			creds := credentials.NewStaticCredentialsProvider(os.Getenv("AWS_ACCESS_KEY_ID"), os.Getenv("AWS_SECRET_ACCESS_KEY"), os.Getenv("AWS_SESSION_TOKEN"))
-			region := os.Getenv("AWS_DEFAULT_REGION")
+			// TODO this should probably be a command-line option that overwrites the env var
```

Comment:
Mind creating an OSD card for this? +1 to making these things configurable

Reply:
Sure thing: https://issues.redhat.com/browse/OSD-9344